ooni/nettests/third_party/psiphon.py

```diff
@@ -88,8 +88,10 @@ def test_psiphon(self):
         log.debug('PsiphonTest.test_psiphon')
 
         self.report['success'] = None
+		self.report['psiphon_installed'] = None
         if not os.path.exists(self.psiphonpath):
             log.err('psiphon path does not exists, is it installed?')
+            self.report['success'] = False
```

Comment:
I would keep this as None, since this allows us to distinguish between a non installed psiphon and an installed, but not functioning psiphon.